source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/index/index.tsx

```diff
@@ -1,4 +1,4 @@
-import * as React from 'react';
+import React from 'react';
```

Comment:
I think I may remove this route in a future PR, but it's handy for right now to know how to interface with the DWKit side of the backend.